quinn-proto/src/connection.rs

```diff
@@ -1304,7 +1310,13 @@ impl Connection {
                             }
                             self.set_params(params)?;
                         }
+
                         self.events.push_back(Event::Connected);
+                        self.endpoint_events.push_back(EndpointEvent::Connected {
+                            handshake_complete: self.side.is_server(),
```

Comment:
The handshake is complete regardless of side here, so this naming is a bit confusing.